Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1104,7 +1104,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
 class UpgradeWorkflow_DD4hepDB(UpgradeWorkflow):
     def setup_(self, step, stepName, stepDict, k, properties):
         if 'Run3' in stepDict[step][k]['--era']:
-            stepDict[stepName][k] = merge([{'--conditions': '120X_mcRun3_2021_realistic_dd4hep_v1', '--geometry': 'DB:Extended', '--procModifiers': 'dd4hep'}, stepDict[step][k]])
+            stepDict[stepName][k] = merge([{'--conditions': '121X_mcRun3_2021_realistic_dd4hep_v1', '--geometry': 'DB:Extended', '--procModifiers': 'dd4hep'}, stepDict[step][k]])
```

Comment:
@tvami @cms-sw/alca-l2 as far as I can see this GlobalTag differs from the regular one (`121X_mcRun3_2021_realistic_v8` as proposed at https://github.com/cms-sw/cmssw/pull/35373) not only in terms of the geometry tags, but there is also the pixel dynamic inefficiency
```console 
$ conddb diff 121X_mcRun3_2021_realistic_dd4hep_v1 121X_mcRun3_2021_realistic_v8
[2021-09-28 09:37:11,986] INFO: Connecting to pro [frontier://PromptProd/cms_conditions]
Record                         Label     pro::121X_mcRun3_2021_realistic_dd4hep_v1 Tag     pro::121X_mcRun3_2021_realistic_v8 Tag    
-----------------------------  --------  ------------------------------------------------  ---------------------------------------   
CSCRecoDigiParametersRcd       -         CSCRECODIGI_Geometry_120DD4hepV1                  CSCRECODIGI_Geometry_112YV2               
CSCRecoGeometryRcd             -         CSCRECO_Geometry_120DD4hepV1                      CSCRECO_Geometry_112YV2                   
DTRecoGeometryRcd              -         DTRECO_Geometry_120DD4hepV1                       DTRECO_Geometry_112YV2                    
EcalPFRecHitThresholdsRcd      -         EcalPFRecHitThresholds_UL_2018_2e3sig             EcalPFRecHitThresholds_34sigma_TL235      
GEMRecoGeometryRcd             -         GEMRECO_Geometry_120DD4hepV1                      GEMRECO_Geometry_113YV4                   
GeometryFileRcd                Extended  XMLFILE_Geometry_120DD4hepV3_Extended2021_mc      XMLFILE_Geometry_120YV2_Extended2021_mc   
HcalParametersRcd              -         HCALParameters_Geometry_120DD4hepV1               HCALParameters_Geometry_112YV2            
HcalRespCorrsRcd               -         HcalRespCorrs_2021_v2.0_mc                        HcalRespCorrs_2021_v3.0_mc                
IdealGeometryRecord            -         TKRECO_Geometry_120DD4hepV1                       TKRECO_Geometry_120YV2                    
PCaloTowerRcd                  -         CTRECO_Geometry_120DD4hepV1                       CTRECO_Geometry_112YV2                    
PEcalBarrelRcd                 -         EBRECO_Geometry_120DD4hepV1                       EBRECO_Geometry_112YV2                    
PEcalEndcapRcd                 -         EERECO_Geometry_120DD4hepV1                       EERECO_Geometry_112YV2                    
PEcalPreshowerRcd              -         EPRECO_Geometry_120DD4hepV1                       EPRECO_Geometry_112YV2                    
PHcalRcd                       -         HCALRECO_Geometry_120DD4hepV1                     HCALRECO_Geometry_112YV2                  
PTrackerParametersRcd          -         TKParameters_Geometry_120DD4hepV1                 TKParameters_Geometry_112YV2              
PZdcRcd                        -         ZDCRECO_Geometry_120DD4hepV1                      ZDCRECO_Geometry_112YV2                   
RPCRecoGeometryRcd             -         RPCRECO_Geometry_120DD4hepV1                      RPCRECO_Geometry_112YV2                   
SiPixelDynamicInefficiencyRcd  -         SiPixelDynamicInefficiency_PhaseI_Run3Studies_v2  SiPixelDynamicInefficiency_PhaseI_v9      
```
is this intended? 
Also given that presumably you want to keep it continuously on par with the regular one excepted the geometry tags, can you consider the `autoCondModifiers` mechanism, see:

https://github.com/cms-sw/cmssw/blob/8d0739291b210d29c0dc374f38aa403842a8e9b9/Configuration/AlCa/python/autoCondModifiers.py#L1-L32